lib/index.js

```diff
@@ -1,7 +1,11 @@
 const TestConstructor = Symbol('testConstructor');
 const AssertionConstructor = Symbol('assertionConstructor');
 
-const defaultFormatter = result => `${result.msg} -> ${result.passed ? 'ok' : 'failed'}`;
+const showActualAndExpected = result => `\n ACTUAL: ${result.actual} \n EXPECTED: ${result.expected}`;
```

Comment:
Perhaps formatActualAndExpected is a better name, because 'show' is for talking about printing or showing something on the screen.

Reply:
Your right! I'll change it.